src/index.js

```diff
@@ -10,6 +10,7 @@ class HtmlWebpackNewRelicPlugin {
         applicationID: undefined,
         license: undefined,
         trustKey: undefined,
+        beacon: undefined,
```

Comment:
Wouldn't the default need to be `bam-cell.nr-data.net` for backward-compatibility?